CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+# Contributing
+
+## License
+
+Intel Project for LLVM\* technology is licensed under the terms of the Apache
```

Comment:
This is copy-pasted wording from intel/llvm project. Could you please fix it so it's relevant to this project.

Reply:
done